SU2_CFD/src/numerics_direct_turbulent.cpp

```diff
@@ -534,6 +534,14 @@ void CSourcePieceWise_TurbSA::ComputeResidual(su2double *val_residual, su2double
 //  AD::SetPreaccIn(TurbVar_i[0]);
 //  AD::SetPreaccIn(TurbVar_Grad_i[0], nDim);
 //  AD::SetPreaccIn(Volume); AD::SetPreaccIn(dist_i);
+  
+//Added by Cakmakcioglu, for BC Transition Model 10/30/2106
```

Comment:
Dr. Albring, thank you very much for your feedback. This is certainly my bad, as I have no previous experience with github, and did not know that you can see my changes that easily! This is why I have added these comment lines, thought that it would be helpful for others to keep track of the additions that I made. I apologize for that. I certainly know that you always appreciate all the contributions to the code. 
The comment lines are deleted.